stable_baselines3/common/sb2_compat/rmsprop_tf_like.py

```diff
@@ -42,7 +44,16 @@ class RMSpropTFLike(Optimizer):
 
     """
 
-    def __init__(self, params, lr=1e-2, alpha=0.99, eps=1e-8, weight_decay=0, momentum=0, centered=False):
+    def __init__(
```

Comment:
Could clean the typing info from docstrings at the same time?

Reply:
yes, I did not know for that one, as it is a clone of PyTorch code. If we want to compare it in future version.
But  will do, so we have only one codestyle.

Reply:
Ah right, good point. It is good at is :)